### PR TITLE
add ExAbsHourMull, ExAbsDayMull uplink commands

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/commands/uplink/ExAbsDayMul.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        for ( let channelIndex = 0; channelIndex <= maxChannel; ++channelIndex ) {
+            // IPK_${channelIndex}
+            const pulseCoefficient = buffer.getUint8();
+            // day value
+            value = buffer.getExtendedValue();
+            channels.push({
+                value,
+                pulseCoefficient,
+                index: channelIndex,
+                meterValue: this.getMeterValue(value, pulseCoefficient)
```
Comment (DarkPark) [inline src/commands/uplink/ExAbsDayMul.ts:55]:
```suggestion
                meterValue: value * pulseCoefficient
```
?

File: src/commands/uplink/ExAbsHourMul.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    toBytes (): Uint8Array {
+        const buffer = new CommandBinaryBuffer(COMMAND_BODY_MAX_SIZE, false);
+        const {channels} = this.parameters;
+
+        const {time} = channels[0];
+        const realDate = getDateFromSeconds(time);
+        const hour = realDate.getUTCHours();
+        let hourAmount = channels[0].diff.length;
+
+        if ( hourAmount === 1 ) {
```
Comment (DarkPark) [inline src/commands/uplink/ExAbsHourMul.ts:97]: would be nice to explain

Comment (DeadByElpy) [reply src/commands/uplink/ExAbsHourMul.ts:98]: todo added, all special places will be documented and linked

File: src/commands/uplink/ExAbsDayMul.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/commands/uplink/ExAbsDayMul.ts]: fixed

File: src/commands/uplink/ExAbsHourMul.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+        for ( let channelIndex = 0; channelIndex <= maxChannel; ++channelIndex ) {
+            // IPK_${channelIndex}
+            const pulseCoefficient = buffer.getUint8();
+            // decode hour value for channel
+            const pulseValue = buffer.getExtendedValue();
+            counterDate.setTime(date.getTime());
+
+            const diff: Array<any> = [];
+            const channel = {
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/commands/uplink/ExAbsHourMul.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**